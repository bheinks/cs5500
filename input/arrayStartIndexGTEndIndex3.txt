(* 
   Start index must be less than or equal to end index of array
*)

program p;

var A : array [-5..-10] of integer;

begin
  A[-5] := 5
end.

