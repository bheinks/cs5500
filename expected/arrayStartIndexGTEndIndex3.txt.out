Line 7: Start index must be less than or equal to end index of array
